esmvaltool/cmor/_fixes/CMIP6/BCC_ESM1.py

```diff
@@ -0,0 +1,65 @@
+# pylint: disable=invalid-name, no-self-use, too-few-public-methods
+"""Fixes for BCC-ESM1."""
+import numpy as np
+import iris
+
+from ..fix import Fix
+
+
+class allvars(Fix):
+    """Common fixes to all vars"""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Fixes error in time coordinate, sometimes contains trailing zeros
+
+        Parameters
+        ----------
+        cube: iris.cube.CubeList
+
+        Returns
+        -------
+        iris.cube.CubeList
+
+        """
+        for cube in cubes:
+            try:
```

Comment:
Try to keep the amount of code between `try` and `except` to the mininum, preferably one line, this improves readability and decreases the chance of accidentally catching an exception raised from a different point than you are expecting.